gapis/perfetto/android/trace.go

```diff
@@ -71,8 +71,8 @@ func setupVulkanLayers(ctx context.Context, d adb.Device, packageName string, ab
 	return cleanup, nil
 }
 
-func setupRenderStagesEnvironment(ctx context.Context, d adb.Device, packageName string, perfettoConfig *perfetto_pb.TraceConfig, abi *device.ABI, layers []string) (app.Cleanup, error) {
-	if !hasRenderStageEnabled(perfettoConfig) {
+func SetupRenderStagesEnvironment(ctx context.Context, d adb.Device, packageName string, hasRenderStages bool, abi *device.ABI, layers []string) (app.Cleanup, error) {
```

Comment:
Since you are making this function public, please rename it. It's a bit silly that it says "do X" and takes a boolean that controls whether "do do X"... Maybe `SetupProfileLayers`

Also, since it's now public, it needs a doc comment.